src/libraries/System.Private.CoreLib/src/System/Globalization/OrdinalCasing.Icu.cs

```diff
@@ -17,7 +17,7 @@ internal static partial class OrdinalCasing
         // s_basicLatin is covering the casing for the Basic Latin & C0 Controls range.
         // we are not lazy initializing this range because it is the most common used range and we'll cache it anyway very early.
         private static readonly ushort[] s_basicLatin =
```

Comment:
Why isn't this one a `ROSpan<ushort>` instead? It doesn't look like its returned anywhere and is instead only used as part of a lookup

Reply:
Because it's stored into a ushort[][] later in the file:
https://github.com/dotnet/runtime/blob/954eb2bc9b0bd9659761b3e0a30df526c3d67a2e/src/libraries/System.Private.CoreLib/src/System/Globalization/OrdinalCasing.Icu.cs#L44-L49